packages/zoltan2/src/algorithms/connectivity/Zoltan2_VtxLabel.hpp

```diff
@@ -0,0 +1,378 @@
+#include "Tpetra_Core.hpp"
+#include "Teuchos_RCP.hpp"
+#include "Tpetra_Import.hpp"
+#include "Tpetra_FEMultiVector.hpp"
+#include "Tpetra_MultiVector_decl.hpp"
+#include "Tpetra_MultiVector_def.hpp"
+
+#include <string>
+#include <sstream>
+#include <iostream>
+#include <queue>
+#include <vector>
+
+#ifndef ZOLTAN2_VTXLABEL_
+#define ZOLTAN2_VTXLABEL_
+
+namespace Zoltan2{
+        
+        template<typename MAP>
+        class iceSheetPropagation;
+	//Graph datastructure that represents the local
+	//graph in ice sheet propagation problems
+        template<typename lno_t>
+	class icePropGraph {
+          public:
+            //n represents the number of vertices in the local graph
+            size_t n;
```

Comment:
nVtx and nEdge would be clearer and more searchable than n and m.

Reply:
I have made this change.